web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -369,11 +369,13 @@ def latest_distinct_program_families_dict(program_families):
     program_families_dict = {}
     for program_family in program_families:
         location, created_at = program_family[0], program_family[1]
+        user_created_at = program_family[2]
         if location in program_families_dict.keys():
+            created_at = (created_at or user_created_at)
```

Comment:
could set the created_at variable up a level and eliminate the need for comparing ```(created_at or user_created_at)``` again in the else statement?